rlclientlib/dedup.cc

```diff
@@ -232,7 +232,8 @@ int action_dict_builder::finalize(generic_event& evt, api_status* status)
     now,
     generic_event::payload_type_t::PayloadType_DedupInfo,
     std::move(payload),
-    content_type);
+    content_type, 
+    evt.get_app_id());
```

Comment:
@kumpera if you can please review this code. I could not manually test this as it is not enabled on the trainer yet.

Reply:
There are unit tests for dedup, you should be able to expand them to check for app-id

Reply:
The test name is action_dict_builder in dedup_test.cc:142

Reply:
dedup_test.cc are not currently added to unit test and some of the tests are failing when I tested on the master branch. Created a issue. Added note to check for app_id in the test. 
https://github.com/VowpalWabbit/reinforcement_learning/issues/348